src/implementations/vanilla/components/carousel/carousel.js

```diff
@@ -31,14 +31,17 @@ export class Carousel {
   constructor(
     element,
     {
-      toggleSelector = '.ecl-carousel__toggle',
+      playSelector = '.ecl-carousel__play',
+      pauseSelector = '.ecl-carousel__pause',
```

Comment:
not sure why we use css class selector here instead of data attributes as on the other components, but that would require some quite big changes...
Probably for next major version of this component

Reply:
I think I wanted to lighten the markup in a way... but indeed we'll have to align this as other components